network/wsPeer.go

```diff
@@ -863,11 +847,19 @@ func (wp *wsPeer) writeLoopSendMsg(msg sendMessage) disconnectReason {
 		return disconnectWriteError
 	}
 	wp.lastPacketTime.Store(time.Now().UnixNano())
-	networkSentBytesTotal.AddUint64(uint64(len(msg.data)), nil)
-	networkSentBytesByTag.Add(string(tag), uint64(len(msg.data)))
-	networkMessageSentTotal.AddUint64(1, nil)
-	networkMessageSentByTag.Add(string(tag), 1)
-	networkMessageQueueMicrosTotal.AddUint64(uint64(time.Now().Sub(msg.peerEnqueued).Nanoseconds()/1000), nil)
+	if wp.peerType == peerTypeWs {
+		networkSentBytesTotal.AddUint64(uint64(len(msg.data)), nil)
+		networkSentBytesByTag.Add(string(tag), uint64(len(msg.data)))
+		networkMessageSentTotal.AddUint64(1, nil)
+		networkMessageSentByTag.Add(string(tag), 1)
+	} else {
+		networkP2PSentBytesTotal.AddUint64(uint64(len(msg.data)), nil)
+		networkP2PSentBytesByTag.Add(string(tag), uint64(len(msg.data)))
+		networkP2PMessageSentTotal.AddUint64(1, nil)
+		networkP2PMessageSentByTag.Add(string(tag), 1)
+
+	}
+	networkMessageQueueMicrosTotal.AddUint64(uint64(time.Since(msg.peerEnqueued).Nanoseconds()/1000), nil)
```

Comment:
`peerEnqueued` is set in `wsPeer.writeNonBlockMsgs` called from `msgBroadcaster.innerBroadcast`. Since there is no duplicate peers in hybrid mode it does not look it measures the same think twice.
splitting it would give an insight to p2p timing in hybrid mode but p2p has its own metric (that we do not expose b/c it is a histogram type). Maybe just to move under the peerType == ws peer ?

Reply:
Or keep it as is and expect it to have count latency for both p2p and ws peers in hybrid mode

Reply:
Added `networkP2PMessageQueueMicrosTotal`